### PR TITLE
Installing dependencies and basic setup

### PR DESCRIPTION
**Pull request by girlpluscomputer**

### CODE REVIEW

File: src/global-style.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { createGlobalStyle } from "styled-components";
+
+const GlobalStyle = createGlobalStyle`
+    *{
```
Comment (arathjz) [inline src/global-style.js]: Add a space between * { please

File: src/global-style.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+        box-sizing: border-box;
+        margin: 0;
+        padding: 0;
+    }
+
```
Comment (arathjz) [inline src/global-style.js]: Remove this line break pls

File: src/index.js [hunk 3]
```diff
@@ -1,6 +1,14 @@
 import React from "react";
 import ReactDOM from "react-dom";
-
+import { ThemeProvider } from "styled-components";
```
Comment (arathjz) [inline src/index.js]: Add a line break here pls
The order of the imports must be

- Third party dependencies (Ex. import  react from 'react')
- Absolute dependencies (Ex, import { Loader } from 'components'
- Own dependencies (Ex. import View from './View'

File: src/theme.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+const theme = {
```
Comment (arathjz) [inline src/theme.js]: Add the media file, so you can handle different screen sizes

You can find an example wether in the example code I provided you or this link https://medium.com/@samuelresua/easy-media-queries-in-styled-components-690b78f50053

File: src/global-style.js (hunk 1 above)
Comment (arathjz) [reply src/global-style.js]: Install a plugin in vscode called Eslint please, it will help you to keep your code clean

Comment (girlpluscomputer) [reply src/global-style.js]: Done

File: src/theme.js (hunk 4 above)
Comment (girlpluscomputer) [reply src/theme.js]: Done

File: src/global-style.js (hunk 2 above)
Comment (girlpluscomputer) [reply src/global-style.js]: Done

**Pull request merged**